src/mmw/js/src/analyze/views.js

```diff
@@ -57,6 +57,33 @@ var ResultsView = Marionette.LayoutView.extend({
             newProjectUrl = '/project/new/' + modelPackageName,
             projectUrl = '/project';
 
+        if (modelPackageName === 'gwlfe') {
+            var analysisResults = JSON.parse(App.getAnalyzeCollection()
+                                                .findWhere({taskName: 'analyze'})
+                                                .get('result'));
+
+            var landResults = _.find(analysisResults, function(element) {
+                return element.name === 'land';
+            });
+
+            if (landResults) {
+                var landAreaResults = landResults.categories;
+
+                var landCoverTotal = _.reduce(_.filter(landAreaResults,
```

Comment:
Prefer `_.sum(_.map())` over `_.reduce(_.filter())` which would be easier to follow.